ibm/service/satellite/data_source_ibm_satellite_location.go

```diff
@@ -36,6 +36,12 @@ func DataSourceIBMSatelliteLocation() *schema.Resource {
 				Computed:    true,
 				Description: "A description of the new Satellite location",
 			},
+			"coreos_enabled": {
```

Comment:
in datasource the field is only computed ...we are just setting the using d.Set(XXX).
so optional needs to removed
```
			"coreos_enabled": {
				Type:        schema.TypeBool,
				Computed:    true,
				Description: "If Red Hat CoreOS features are enabled within the Satellite location",
			},
			```

Reply:
Add the attribute in respective datasource doc also under Attribute Section
website/docs/d/satellite_location.html.markdown file

Reply:
Ok, thanks for the clarification. Updated.